util/fips-checksums.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
+
+HERE=`dirname $0`
+
+for f in "$@"; do
+    case "$f" in
+        *.c | *.h )
+            cat "$f" \
+                | $HERE/lang-compress.pl 'C' \
+                | unifdef -DFIPS_MODE=1 \
+                | openssl sha256 -r \
+                | sed -e "s| \\*stdin| *$f|"
```

Comment:
Why `*$f` and not just `$f`?

Reply:
Because that's the output that `openssl dgst -r` gives...  however, on the reading the sha1sum manual, the `*` should really be a space, since these are text files (and `openssl dgst` actually does that wrong)

Reply:
I've made a fix to get it right